### PR TITLE
Fixed the openiconic link

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: src/Templates/src/templates/maui-blazor/wwwroot/css/open-iconic/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-[Open Iconic v1.1.1](http://useiconic.com/open)
+[Open Iconic v1.1.1](https://github.com/iconic/open-iconic)
 ===========
 
-### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](http://useiconic.com/open#icons)
+### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://github.com/iconic/open-iconic)
```
Comment (danroth27) [inline src/Templates/src/templates/maui-blazor/wwwroot/css/open-iconic/README.md:4]:
```suggestion
### Open Iconic is the open source sibling of [Iconic](https://github.com/iconic/open-iconic). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://github.com/iconic/open-iconic)
```

Comment (mkArtakMSFT) [reply src/Templates/src/templates/maui-blazor/wwwroot/css/open-iconic/README.md:4]: Thanks for catching this!

**Review by danroth27 — APPROVED** [review]
LGTM

**Review by danroth27 — APPROVED** [review]
LGTM

**Review by jsuarezruiz — APPROVED** [review]

**Review by rmarinho — APPROVED** [review]

**Review by surayya-MS — APPROVED** [review]

**Pull request merged**